clang/test/CIR/CodeGen/inlineAttr.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -O2 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s -check-prefix=CIR
+// RUN: %clang_cc1 -O2 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-llvm %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s -check-prefix=LLVM
+
+
+inline int s0(int a, int b) {
+  int x = a + b;
+  return x;
+}
+
+__attribute__((noinline))
+int s1(int a, int b) {
+  return s0(a,b);
+}
+
+__attribute__((always_inline))
+int s2(int a, int b) {
+  return s0(a,b);
+}
+
+int s3(int a, int b) {
+  int x = a + b;
+  return x;
+}
+
+
+// CIR:   cir.func linkonce_odr @_Z2s0ii(%arg0:{{.*}}, %arg1:{{.*}} -> {{.*}} extra_attributes {#cir.inline<hint>}
+// CIR:   cir.func @_Z2s1ii(%arg0:{{.*}}, %arg1:{{.*}} -> {{.*}} extra_attributes {#cir.inline<no>}
```

Comment:
To ilustrate my previous comments, I'd like this one to look like:
```
// CIR:   cir.func @_Z2s1ii(%arg0:{{.*}}, %arg1:{{.*}} -> {{.*}} extra({#cir.inline<no>})
```
The idea is that when we add aliases in the future, this could look like:
```
#extra._Z2s1ii = #cir.extra({#cir.inline<no>})
// CIR:   cir.func @_Z2s1ii(%arg0:{{.*}}, %arg1:{{.*}} -> {{.*}} #extra._Z2s1ii
```